vcpkg.json

```diff
@@ -2,6 +2,17 @@
   "builtin-baseline": "000d1bda1ffa95a73e0b40334fa4103d6f4d3d48",
   "dependencies": [
     "gtest",
-    "zlib"
+    "zlib",
+    {
+      "name": "sdl2",
+      "version>=": "2.30.1",
```

Comment:
2.30.2 has not been released on VCPKG yet ?

Reply:
No, it has not.

Reply:
It's good practice IMHO, as this is what we expect as a minimum so get it "documented" is nice. Also if you have an older vcpkg ports baseline and you suddenly install `2.28.0` instead for example there will be no error (that said we specify the baseline here as well, so this particular issue won't happen).